src/components/Footer.jsx

```diff
@@ -0,0 +1,32 @@
+import Image from "next/image";
+import AILogo from "@/public/images/aitransparentlogo.png";
```

Comment:
dont import images like this, while they are in public they can be referenced directly
for more information: https://nextjs.org/docs/app/building-your-application/optimizing/static-assets